packages/docusaurus-theme-classic/src/theme/NotFound.tsx

```diff
@@ -5,10 +5,10 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
+import React, {ReactElement} from 'react';
 import Layout from '@theme/Layout';
 
-function NotFound() {
+function NotFound(): ReactElement {
```

Comment:
One concern I have is that in the past, I have had a lot of issues with typing react components, because it seems like there are a million different types for components, e.g. `React.Component`, `React.Element`, and `React.ReactElement` (never actually seen this one before).  Might this potentially cause type mismatches?

Reply:
I just added this to show Babel type stripping works. I don't think we will keep it in final release.